scout/server/blueprints/variants/controllers.py

```diff
@@ -49,28 +50,46 @@ def variants(store, institute_obj, case_obj, variants_query, page=1, per_page=50
         overlapping_svs = [sv for sv in store.overlapping(variant_obj)]
         variant_obj['overlapping'] = overlapping_svs or None
 
+        # show previous classifications for research variants,
+        is_research = False
+        if variant_obj['variant_type'] == 'research':
+            is_research = True
+
         # Get all previous ACMG evalautions of the variant
         evaluations = []
         for evaluation_obj in store.get_evaluations(variant_obj):
             classification = evaluation_obj['classification']
             # Only show pathogenic/likely pathogenic from other cases on variants page
             if evaluation_obj['case_id'] == case_obj['_id']:
-                continue
+                if not is_research:
```

Comment:
Could this if be put before, so the loop doesn't even start?

Reply:
Hm, not easily without making two loops? We still want to go through previous evals for both research and non research.

Reply:
I'll add in a a couple of comments that may make it more clear.